### PR TITLE
feat: automate greetings using github bot

### PR DESCRIPTION
**Pull request by dakshsinghrathore**
This PR fixes the issue #31   , kindly review it and merge. @dvjsharma 

You need to give `read and write permission` to the bot to make comments hence kindly go to the repo  `settings/actions/General`  and under  `Workflow Permissions`  grant  `Read and write` permissions and then `save`.

like this ⬇️

image

### CODE REVIEW

**Review by dvjsharma — CHANGES_REQUESTED** [review]
Minor changes needed

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: 'Greetings'
+
+on:
+  fork:
+  push:
+    branches: [main]
```
Comment (dvjsharma) [inline .github/workflows/greetings.yml:6]: Replace with `master`

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Congratulations, @${{ github.actor }}! 🎉 Thank you for creating your issue. Your contribution is greatly appreciated and we look forward to working with you to resolve the issue. Keep up the great work!
+          pr-message: |
+            Great job, @${{ github.actor }}! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.
+          footer: 'We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/dvjsharma/Drawn2Shoe/blob/master/CONTRIBUTING.md)'
```
Comment (dvjsharma) [inline .github/workflows/greetings.yml:24]: Grammatical errors, please check with AI models.

**Review by dvjsharma — CHANGES_REQUESTED** [review]
Minor changes needed

**Comment by dvjsharma** [comment]
Any updates @dakshsinghrathore?

**Comment by dakshsinghrathore** [comment]
@dvjsharma have made the changes kindly review

**Comment by dvjsharma** [comment]
@dakshsinghrathore Can't see any. Please re-check

**Comment by dvjsharma** [comment]
Closing since no response. Please re-open if necessary.

**Pull request closed without merging**